src/components/BadgesFromPkg.tsx

```diff
@@ -27,6 +28,7 @@ export const badgeComponents = {
   githubIssues: GithubIssuesBadge,
   jsxReadme: JsxReadmeBadge,
   semanticRelease: SemanticReleaseBadge,
+  npmBundleSize: NpmBundleSizeBadge,
```

Comment:
Please move directly after the NpmDownloadsBadge

Reply:
Sure thing! I have pushed a new commit that fixes that.